mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -50,7 +50,11 @@ export class ExperimentPage extends Component {
     const orderBy = ExperimentPage.getOrderByExpr(orderByKey, orderByAsc);
     const viewType = lifecycleFilterToRunViewType(lifecycleFilter);
 
-    this.props.getExperimentApi(experimentId, this.getExperimentRequestId);
+    this.props
+      .getExperimentApi(experimentId, this.getExperimentRequestId)
+      .catch((e) => {
```

Comment:
I noticed that we are missing promise rejection in many places other than this. Let's create a follow-up to have those places adjusted as well.